### ISSUE TITLE
ngChange event fires when editor is initially populated

### ISSUE THREAD

**Issue by wmenge** [zanemayo/angular-trix-editor#1]
Cool directive! Was checking out the editor and was just thinking of building my first directive. 

With a normal input field, the ngChange directive doesn't fire an event when the field is initially populated with ngModel. In the case of the trix editor, this is not the case. I'll try to create a plunker or something to demonstrate later.

**Comment by zanemayo** [comment]
Thank you for the report. 
I've just added Jasmine and Karma for testing. I will reproduce the bug and fix it soon hopefully

**Comment by wmenge** [comment]
I've done some testing: It seems the trix-change event is being fired when trix modifies it's input value, e.g. when `Test` becomes `<div>Test</div>`.

I've also tested the trix editor with forms integration:

```
<form>
    <input id="x" value="Test" type="text" name="content">
    <trix-editor input="x"></trix-editor>
</form>

<script>
    document.addEventListener("trix-change", function(event) {
        console.log('Changed!!!');
    });
</script>
```

In this case, the trix-change event only fires when the user makes a change, not while initially populating. I woult expect the same behaviour with this wrapper.

**Comment by wmenge** [comment]
Closed with pull request #2
